e2e_tests/config.py

```diff
@@ -14,3 +14,8 @@
 TEST_ACCOUNT_CLIENT_ID: str = config("TEST_ACCOUNT_CLIENT_ID", default="")
 TEST_ACCOUNT_CLIENT_SECRET: str = config("TEST_ACCOUNT_CLIENT_SECRET", default="")
 TEST_WORKSPACE_APP_ID: str = config("TEST_WORKSPACE_APP_ID", default="")
+
+# Perf test env vars - set these in private.env if you want to run perf tests and use an existing
```

Comment:
Should say /templates/core/.env. private.env will be autogenerated and will wipe your settings. The launch target you created copies the /templates/core/.env into e2e folder and also then pulls in private.env. (Effectively tests read 2 .env files)